### ISSUE TITLE
No support to use list of host in rules ingress 

### ISSUE THREAD

**Issue by jotasixto** [prefapp/prefapp-helm#96]
It would be appropriate to allow host lists in ingress rules, similar to tls host list support.

**Comment by frmadem** [comment]
@jotasixto the new functionality should let render ingress rules such as:

```yaml
{{- define "ingress_multiple.data" -}}

version: {{ include "ph.k8s.version" . }}

name: {{ .Release.Name }}

tls:
  - hosts: ["a.com", "b.com"]
    secretName: foo

hosts:
  - host: foo.com
    rules_path:
    - path: /
      service: {{.Release.Name}}-foo
      port: 80

  - host: foo2.com
    rules_path:
    - path: /
      service: {{.Release.Name}}-foo2
      port: 80


{{- end -}}

{{ include "ph.ingress.render" (include "ingress_multiple.data" . | fromYaml) }}

```

**Comment by frmadem** [comment]
This #97 should resolve the problem @jotasixto

**Comment by jotasixto** [comment]
@frmadem Thanks! I will test it.

**Comment by jotasixto** [comment]
@frmadem 
When doing the tests we found this problem:
```
STDERR:
  Error: Failed to render chart: exit status 1: Error: YAML parse error on netexcloud/charts/integration/templates/ingress.yaml: error converting YAML to JSON: yaml: line 23: did not find expected alphabetic or numeric character
  Use --debug flag to render out invalid YAML
  Error: plugin "diff" exited with error
```
This occurs when we use a wildcard host like `*.pre.test.run`.

**Comment by frmadem** [comment]
> @frmadem
> When doing the tests we found this problem:
> 
> ```
> STDERR:
>   Error: Failed to render chart: exit status 1: Error: YAML parse error on netexcloud/charts/integration/templates/ingress.yaml: error converting YAML to JSON: yaml: line 23: did not find expected alphabetic or numeric character
>   Use --debug flag to render out invalid YAML
>   Error: plugin "diff" exited with error
> ```
> 
> This occurs when we use a wildcard host like `*.pre.test.run`.

@jotasixto did you try to quote it using "?

I tried the following:


```yaml

# values.yaml
ingress_multiple:
  hosts:
    - a.com
    - b.com
    - "*.pre.test.run"
```

And in my template:

```yaml
{{- define "ingress_multiple.data" -}}

version: {{ include "ph.k8s.version" . }}

name: {{ .Release.Name }}

tls:
  - hosts: {{ .Values.ingress_multiple.hosts | toYaml | nindent 6}}
    secretName: foo
...
{{- end -}}
```
And it seems to work:

```yaml

# output 

  tls:


    - hosts:
      - a.com
      - b.com
      - '*.pre.test.run'
      secretName: foo

```

**Comment by gustavoborragan** [comment]
@jotasixto, the problem has been fixed in this #111 

Wildcards can now be used (start of string with asterisk).

**Comment by frmadem** [comment]
Solved on #111
